_build_cfg.yml

```diff
@@ -265,6 +265,10 @@ Topics:
     File: configuring_authentication
   - Name: Syncing Groups With LDAP
     File: syncing_groups_with_ldap
+  - Name: Setting up SSSD for LDAP Failover
+    File: sssd_for_ldap_failover
+  - Name: Configuring Form-Based Authentication
+    File: configuring_form_based_authentication
```

Comment:
and I forgot to say, would it be worth nesting all these LDAP-related topics into their own drop-down here, instead of having 3 or 4 or however many LDAP topics cluttering up the main install_config dir? :smile_cat: